xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/restore.vm

```diff
@@ -110,7 +110,10 @@
   <div class="panel $!panelClass">
     <div class="panel-heading">
       $heading
-      <div class="pull-right"><a class="panel-collapse-carret" role="button" data-toggle="collapse" href="#$panelId" aria-expanded="false" aria-controls="$panelId"><span class="caret"></span></a></div>
+      <div class="pull-right"><a class="panel-collapse-carret" role="button" data-toggle="collapse" href="#$panelId" aria-expanded="false" aria-controls="$panelId">
+        <span class="caret"></span>
+        <label class='sr-only'>$escapetool.xml($services.localization.render('core.delete.affectChildren.dropDown.label'))</label>
```

Comment:
Reference: https://html.spec.whatwg.org/#interactive-content

Reply:
For anchors, there's no need to use a label, we can just put a span with the `sr-only` class inside of it. Its accessible name will follow its content.
![displayOfAnAnchorWithASpanAsLabel](https://github.com/xwiki/xwiki-platform/assets/28761965/260ef2ee-60ee-4231-8382-4b8a35c95cf4)

Reply:
Yes, but why is the translation name displayed instead of the translated string in your screenshot? Is this a bug or just due to how you generated the screenshot?

Reply:
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/35baf0c4-27db-407d-b0a1-5f337e7bc263)
Notice the correct name of the element in the Accessibility tree (minus the fact that I didn't bother getting the translations on my distrib)

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2215/commits/bb81586d9c450d8dae3644557c480331c3370fe5

Reply:
> Yes, but why is the translation name displayed instead of the translated string in your screenshot? Is this a bug or just due to how you generated the screenshot?

It's just due to how I generated the screenshot, sorry for the imprecision. I did not get the translations on my live test distribution.